sdk/python/kfp/az.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2019 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def use_az_secret(secret_name='azcreds'):
```

Comment:
Do you mind adding a basic unit test to cover this method?

Reply:
I'm not fully sure how to test this.
How would you unit-test this code?

Reply:
OK. I guess we can test that after calling `use_az_secret` the `ContainerOp` has 4 env variables with specific names.

Reply:
Good place for the tests: https://github.com/kubeflow/pipelines/tree/master/sdk/python/tests/dsl / `test_azure_extensions.py`